scripts/generator.py

```diff
@@ -41,6 +42,9 @@
 def main() -> None:
     args = argument_parser()
 
+    if not args.semconv_version:
+        raise KeyError("OTel Semantic Conventions version has not been provided as a config option '--semconv-version'")
```

Comment:
should the default be the last one? we can detect it through git and skip manually selecting it
I think for this purpose we need always to adhere to the last available version

Reply:
I'd rather keep that a manual step, or use something like dependabot to update it but not updating it automatically, that has some risks.

Reply:
we are using there only schema, what risks do you have in mind?
we are adding a lot of new fields into each release but most probably we would need to update mapping anyway and can update the version, so this comment is not blocking.